ingestion_tools/dataset_configs/submission_template.yaml

```diff
@@ -0,0 +1,46 @@
+dataset:
+  dataset_identifier: REQUIRED #(ex: 10004)
+annotations:
```

Comment:
Should the annotation be mapped to a tomogram or voxel spacing now?

Reply:
Annotations should be mapped to tomograms.

Reply:
They're currently mapped to voxel spacings -- should they be mapped directly to tomograms, or to voxel spacing + alignement?